added a test function that does nothing called test_fun

John-Polo opened the pull request.

John-Polo committed:
added a test function that does nothing called test_fun

wenzeslaus commented:
John, thanks for creating this PR. Perhaps you are already aware of that, but there are two immediate problems:

1. The Python file with your change is renamed file from Anna rather than a new file. It also misses the file extension `.py`. How this is expressed in the GitHub interface is:

```
 activities/activity_anna_petrasova.py → activities/activity_john_polo
```

You will need create a new file and tell Git about it, e.g., using `git add`.

This is also the cause of the error in the Render activities test (`.../activity_anna_petrasova.py': [Errno 2] No such file or directory`).

2. You made the PR from your main branch rather than a new dedicated branch. That's not an issue for your first PR, but you will need to make some fix before you do another PR. If all your work is saved elsewhere, deleting your fork and the local repo is [a fine fix](https://m.xkcd.com/1597/).

wenzeslaus commented:
This also made me realize that some tests are not working as this should have been reported more clearly by the checks. Fixed in #11. Thanks again.

John-Polo committed:
Create config_johnpolo.json

John-Polo committed:
Delete activity_john_polo

John-Polo committed:
Create activity_johnpolo.py

This is to calculate the profc: profile curvature (curvature intersecting with the plane defined by Z axis and maximum gradient direction). It is based on "Additional" from https://grasswiki.osgeo.org/wiki/Using_GRASS_GIS_through_Python_and_tangible_interfaces_(workshop_at_FOSS4G_NA_2016)
Positive values describe convex profile curvature, negative values concave profile curvature.

John-Polo committed:
Delete config_anna_petrasova.json

John-Polo committed:
Update activity_johnpolo.py

John-Polo committed:
Update config_johnpolo.json

John-Polo committed:
Update activity_johnpolo.py

remove env from the functions to fix PR error.
